docs/content_management/field_types/type_and_value.md

```diff
@@ -62,7 +62,13 @@ To generate Content item name or URL alias the Field Type name must be a part of
 
 ## Value handling
 
-A Field Type needs to deal with the custom value format provided by it. In order for the public API to work properly, it delegates working with such custom Field values to the corresponding Field Type. The `Ibexa\Core\FieldType\FieldType` interface therefore provides the following methods:
+A Field Type needs to deal with the custom value format provided by it. In order for the public API to work properly, it delegates working with such custom Field values to the corresponding Field Type. The `Ibexa\Core\FieldType\FieldType` interface therefore provides the following methods.
+
+!!! caution "Simple hash values"
+
+    In [[= product_name =]], a simple hash value always means an array of scalar values and/or nested arrays of scalar values.
+    To avoid issues with format conversion, do not use objects inside the simple hash values.
+
```

Comment:
This part is rather related to "Serialization" sub-section than to the methods mentioned directly below ;)